engine/src/main/java/org/destinationsol/game/ObjectManager.java

```diff
@@ -356,7 +360,8 @@ public void addFarObjNow(FarObject fo) {
         return myFarPorts;
     }
 
-    public void dispose() {
+    @Override
+    public void close() throws Exception {
         myWorld.dispose();
```

Comment:
This doesn't appear to be called anywhere. Might this cause a memory leak?

Reply:
Autoclosable is automatically called when the context is freed